alfresco-transform-libreoffice/alfresco-transform-libreoffice/src/main/java/org/alfresco/transformer/executors/LibreOfficeJavaExecutor.java

```diff
@@ -53,12 +53,13 @@
     private static final Logger logger = LoggerFactory.getLogger(LibreOfficeJavaExecutor.class);
 
     private static final int JODCONVERTER_TRANSFORMATION_ERROR_CODE = 3088;
-    private static final String OFFICE_HOME = "/opt/libreoffice6.1";
+    private static String OFFICE_HOME;
```

Comment:
Could/should this be "final" (rather than "static") ?

Reply:
It can't be final in this case as final vars need to be initialised and then won't be able to reassigned it with the new value at runtime.

Reply:
Same reply as above ;-)

Reply:
The method accessing this variable is static

Reply:
OK, thanks. Trivial - could rename to LIBREOFFICE_HOME for consistency with env var ?